src/utils/sheets.js

```diff
@@ -0,0 +1,47 @@
+import c from '../config'
+import {google} from 'googleapis'
+
+// Authentication
+// Google Sheet must give read permission to the service account in c.google.email
+const scopes = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+const key = Buffer.from(c.google.key, 'base64').toString()
+const auth = new google.auth.JWT(c.google.email, null, key, scopes)
+const sheetsApi = google.sheets({version: 'v4', auth})
+
+/**
+ * Loads data from Google Sheets
+ * Returns an array of objects (rows) indexed by column headers (first row) 
+ * Called from contract template
+ * @param {*} people
+ * @param {spreadsheetId, sheetId, range, idSheetsColumn, idEMSField} sheet: Google sheet and range identifier, fields to join with EMS
+ */
+export const loadSheetData = async (people, sheet) => {
+  const personIDs = people.map((person) => person[sheet.idEMSField])
+
+  let [header, ...values] = await getValues(sheet.spreadsheetId, sheet.range)
+
+  // Index by column headers,
+  // filter rows in the array people - from the original request
+  values = values
```

Comment:
It is a good practice not to overshadow values, so it should be renamed to something like "filteredValues" (or better). Also, make `header` and `values` constants.
```suggestion
  const filteredValues = values
```